src/test/java/edu/kis/powp/jobs2d/events/SelectLoadMacroListener.java

```diff
@@ -0,0 +1,26 @@
+package edu.kis.powp.jobs2d.events;
+
+import edu.kis.powp.jobs2d.command.DriverCommand;
+import edu.kis.powp.jobs2d.drivers.DriverManager;
+import edu.kis.powp.jobs2d.features.CommandsFeature;
+import edu.kis.powp.jobs2d.features.MacroFeature;
+
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+
+public class SelectLoadMacroListener implements ActionListener {
+    private DriverManager driverManager;
+
+    public SelectLoadMacroListener(DriverManager driverManager) {
+        this.driverManager = driverManager;
+    }
+
+    @Override
+    public void actionPerformed(ActionEvent e) {
+        CommandsFeature.getDriverCommandManager().setCurrentCommand(MacroFeature.getMacroDriverAdapter().getDriverCommandList(), "Macro");
+        driverManager.setCurrentDriver(MacroFeature.getMacroDriverAdapter().getDriver());
+        DriverCommand driverCommand = CommandsFeature.getDriverCommandManager().getCurrentCommand();
+        driverCommand.execute(driverManager.getCurrentDriver());
```

Comment:
If name changes from 'run' to 'load' then the logic should also be updated. Please remove it and test if it works using "Run command" from menu.